GHzDACs/ghz_fpga_server.py

```diff
@@ -1631,16 +1655,100 @@ def dac_run_sram(self, c, data, loop=False, blockDelay=0):
         """
         if not len(data):
             return
-        
-        if loop:
-            # make sure data is at least 20 words long by repeating it
-            data *= (20-1)/len(data) + 1
-        else:
-            # Set data at least 20 words long by repeating first value
+
+        if len(data) < 20:
+            # pad out to 20 with zeros
```

Comment:
This is different behavior from before; padding with zeros will definitely change the looped waveform, and previously the padding used the first SRAM value, which I believe was the output value when the board idled after running the SRAM. Do we rely on the old behavior? What was the reason for the change? Do we need to handle this differently for JT versus non-JT boards?